db/schema.rb

```diff
@@ -10,24 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_11_09_013523) do
+ActiveRecord::Schema.define(version: 2018_11_01_003438) do
```

Comment:
I think you may want to `git pull origin master` to make sure this is up-to-date.